L1Trigger/TrackFindingTracklet/interface/AllProjectionsMemory.h

```diff
@@ -0,0 +1,35 @@
+#ifndef L1Trigger_TrackFindingTracklet_interface_AllProjectionsMemory_h
+#define L1Trigger_TrackFindingTracklet_interface_AllProjectionsMemory_h
+
+#include "L1Trigger/TrackFindingTracklet/interface/MemoryBase.h"
+
+#include <string>
+#include <vector>
+
+namespace Trklet {
```

Comment:
we have changed namespace name to start with lowercase: Trklet => trklet (note, "trklet" rather than "tracklet" to distinguish further and avoid confusion with Tracklet class)